www/index.html

```diff
@@ -397,6 +397,16 @@ <h3 id="community"><a href="#community">Community</a></h3>
                 won't be sent in the query string or body anymore.
             </td>
         </tr>
+        <tr>
+            <td>
+                <a href="https://github.com/grey4owl/htmx-preserve-attr/README.md">
```

Comment:
This link doesn't seem to work, you might want to either point to https://github.com/grey4owl/htmx-preserve-attr or https://github.com/grey4owl/htmx-preserve-attr/blob/main/README.md